### PR TITLE
Stop using OpenCV in examples

### LINKED ISSUE #88: Stop using OpenCV to just reading a JPEG file in the ImageNet example
Current ImageNet example uses the cv2 package to just load JPEG image files. This is overkill.

### PR DESCRIPTION
**Pull request by okuta**
I fixed #88

### CODE REVIEW

File: examples/imagenet/README.md [hunk 1]
```diff
@@ -1,9 +1,5 @@
 # Large Scale ConvNets
 
-## Requirements
-
-- OpenCV 2.4
```
Comment (unnonouno) [inline examples/imagenet/README.md]: We need to add `pillow` instead of `opencv`

File: setup.py [hunk 2]
```diff
@@ -17,6 +17,7 @@
               'chainer.utils'],
     install_requires=['nose',
                       'numpy',
+                      'pillow',
```
Comment (unnonouno) [inline setup.py]: `pillow` is only required for examples. You don't need to add it here.

**Comment by unnonouno** [comment]
We will need to fix the test for examples after this PR. https://github.com/pfnet/chainer-test/blob/master/test_example.sh

File: examples/modelzoo/evaluate_caffe_net.py [hunk 3]
```diff
@@ -92,8 +92,7 @@ def forward(x, t):
 accum_loss = 0
 accum_accuracy = 0
 for path, label in dataset:
-    # TODO(beam2d): Remove dependency on OpenCV
-    image = cv2.imread(path).transpose(2, 0, 1)
+    image = np.asarray(Image.open(path)).transpose(2, 0, 1)[::-1]
```
Comment (unnonouno) [inline examples/modelzoo/evaluate_caffe_net.py]: Is `[2,1,0]` required?

Comment (okuta) [inline examples/modelzoo/evaluate_caffe_net.py]: It uses `[::-1]` instead of `[2,1,0]`

Comment (unnonouno) [inline examples/modelzoo/evaluate_caffe_net.py]: oh, i see.

**Comment by unnonouno** [comment]
LGTM

**Pull request merged**